rover/Process.proto

```diff
@@ -0,0 +1,97 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+syntax = "proto3";
+
+package skywalking.v3;
+
+option java_multiple_files = true;
+option java_package = "org.apache.skywalking.apm.network.rover.process.v3";
+option go_package = "skywalking.apache.org/repo/goapi/collect/rover/process/v3";
+
+import "common/Common.proto";
+
+// Define the Rover detected processes and report them.
+service ProcessService {
+    // Report discovered process in Rover
+    rpc reportProcesses (ProcessList) returns (RoverProbeDownstream) {
+    }
+
+    // Keep the process alive in the backend.
+    rpc keepAlive (ProcessPingPkg) returns (RoverProbeDownstream) {
+    }
+}
+
+message ProcessList {
+    repeated ProcessProperties processes = 1;
+}
+
+message ProcessProperties {
+    // [required] Detect process mode
+    ProcessOwner owner = 1;
+    // [required] Host Process metadata
+    ProcessMetadata metadata = 2;
+    // [optional] Mapping process with other instance
+    // required when the process have agent/sdk installed
+    MappedInstance mappedInstance = 3;
```

Comment:
When you report ProcessProperties, I don't think you have a chance to having this. Because based on your design, this mapping happens at OAP side. This mappedInstance only could retrieve from OAP side. I am not sure why `RoverProbeDownstream` is totally blank? And Instance ID is not enough, you need a process ID(maybe UUID) from OAP for further profiling tasks. Right now, you don't have any ID to assign the task.

And working mechanism(attachment or independent mode) is missing. The OAP should generated instance ID according.

Reply:
Deleted. Rover provides as complete information as possible. If there is a process reported through SDK/Agent, OAP will match the the process between Rover and SDK/Agent